### PR TITLE
Added unit test for cover block render function

### PR DESCRIPTION
**Pull request by akasunil**
## What?

Added unit test for cover block render function.

### CODE REVIEW

**Comment by akasunil** [comment]
@Mamaduka can you please review the PR.

File: phpunit/blocks/render-block-cover-test.php [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+		wp_delete_post( self::$post->ID, true );
+		wp_delete_post( self::$attachment_id, true );
+	}
+
+	/**
+	 * Test render_block_core_cover() method.
+	 *
+	 * @covers ::render_block_core_cover
+	 */
+	public function test_render_block_core_latest_posts() {
```
Comment (draganescu) [inline phpunit/blocks/render-block-cover-test.php]: Should this be `test_render_block_core_cover`?

File: phpunit/blocks/render-block-cover-test.php [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+			'hasParallax'      => true,
+			'isRepeated'       => true,
+		);
+
+		$content = '<div class="wp-block-cover"><span></span><div class="wp-block-cover__inner-container"></div></div>';
+
+		$wp_query->post  = self::$post;
+		$wp_query->posts = array( self::$post );
+
+		$attributes['minHeight'] = '100px';
```
Comment (draganescu) [inline phpunit/blocks/render-block-cover-test.php]: Let's move this to the `$attributes` higher up at line 68.

**Review by draganescu — CHANGES_REQUESTED** [review]
Letf two nits, but by bigger request would be to split into multiple tests as it's a bit confusing reading what we're testing.

File: phpunit/blocks/render-block-cover-test.php (hunk 2 above)
Comment (akasunil) [reply phpunit/blocks/render-block-cover-test.php]: Addressed your feedback. Please review it again.

File: phpunit/blocks/render-block-cover-test.php [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+ */
+
+/**
+ * Tests for the Cover block.
+ *
+ * @group blocks
+ */
+class Tests_Blocks_Render_Cover extends WP_UnitTestCase {
+	/**
+	 * Post object .
```
Comment (draganescu) [inline phpunit/blocks/render-block-cover-test.php]:
```suggestion
	 * Post object.
```

**Review by draganescu — APPROVED** [review]
One nitpick, everything else LGTM

**Comment by akasunil** [comment]
Ready to Merge ?

**Comment by Mamaduka** [comment]
@sunil25393, it looks like PHP Unit tests are failing.

**Comment by akasunil** [comment]
> @sunil25393, it looks like PHP Unit tests are failing.

Can test again, I was calling wrong method of block render.

**Pull request merged**